src/proto/Dependency.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+option java_multiple_files = true;
+option java_package = "com.microsoft.azure.functions.rpc.messages";
+option java_outer_classname = "DependencyProto";
```

Comment:
remove java and go. When the language worker is ready to consume this feature language worker owner will add this.